components/table/basic/src/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const CELL_TYPE = {
+  header: 'th',
+  data: 'td'
+}
+
+const TableBasic = ({contentHead, contentBody, contentFoot}) => {
+  const hasHead = Boolean(contentHead?.length)
+  const hasFoot = Boolean(contentFoot?.length)
+
+  return (
+    <table className="sui-TableBasic sui-TableBasic--noWrap">
```

Comment:
you can declare a `baseClass` and then reuse it everywhere combined with template strings as follows:

```javascript
const baseClass = 'sui-TableBasic'

return (
    <table className={`${baseClass}  ${baseClass}--noWrap`}>
    (...)
```